### DISCUSSION TITLE
Allow setting static access token for WYSIWYG editor in Flows Email operation

### DISCUSSION THREAD

**Discussion by arladmin** [directus/directus#24902] [Feature Requests]
### Describe the Bug

When i try to upload an image in the WYSIWYG editor, in the email's body, in a flow, the file gets successfully uploaded but it doesn't show up in the editor and also doesn't show up in the email, once received by the recipient.

In the editor,  it shows:
<broken image icon>title of the image file

### To Reproduce

Upload an image using the WYSIWYG editor in the email section.

### Errors Shown

Browser: `GET https://my-domain/assets/asset-id  403 (Forbidden)`
Docker console: `You don't have permission to access this.`

### What version of Directus are you using?

9.18.1

### What version of Node.js are you using?

default

### What database are you using?

Postgres 14

### What browser are you using?

Chrome

### How are you deploying Directus?

Docker

**Comment by azrikahar** [comment]
This seems to be the same as #12297, which is technically by design. You can refer to this comment here for more info and how to make it work as expected: https://github.com/directus/directus/issues/12297#issuecomment-1076488908

That being said, the "add a static access token to add to the interface options" step isn't achievable  in terms of the WYSIWYG editor in Email operation, so that should be added.

![chrome_dvVpL3NFkF](https://user-images.githubusercontent.com/42867097/192706114-85102321-9442-4e01-8205-8cfe84353a22.png)

In the meantime, if you don't want to go with the public role route (technically it makes sense since email images are typically served via public CDN from what I understand), you should be able to circumvent this by adding said access token directly in the image URL like so:

![chrome_7TmxkmO18I](https://user-images.githubusercontent.com/42867097/192706052-c6ba2192-4db7-45d7-8d76-81adda3e7c46.png)

Additional note: In case the concern of granting `directus_files` to Public Role is exposing all files, you can actually create a "Public" folder in Files, then only grant permission to files within the Public folder, not all files 👍

**Comment by rijkvanzanten** [comment]
Linear: [ENG-178](https://linear.app/directus/issue/ENG-178/allow-setting-static-access-token-for-wysiwyg-editor-in-flows-email)
